docs/pages/access-controls/access-graph/gitlab.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Discover GitLab Access Patterns with Teleport Policy
+description: Describes how to syncronize GitLab access patterns using Teleport Policy and Access Graph.
+---
+Teleport Policy streamlines and centralizes access management across your entire infrastructure, allowing you to view
+access relationships within seconds. You can see unified, up-to-date relationships and policies between all users, groups,
+and computing resources.
+
+With Teleport Policy's Access Graph, you gain insights into access patterns within your GitLab account. By scanning all
+permissions, users, groups, and projects, it provides a visual representation to help enhance the permission model within
+your GitLab environment. This functionality enables you to answer queries such as:
+
+- What projects are accessible to users?
+- Which users have write permissions to projects?
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise customers.
+
+After logging in to the Teleport UI, navigate to the Management tab. If enabled, Access Graph options can be found
+under the Permission Management section.
+
+## How it works
+
+Access Graph synchronizes various GitLab resources, including users, projects and groups.
+These resources are then visualized using the graph representation detailed in the
+[Access Graph page](../teleport-policy/getting-started-policy.mdx).
+
+The importing process involves two primary steps:
+
+### Polling GitLab APIs
+
+The Teleport cluster continuously scans the configured GitLab accounts and retrieves the following resources:
+
+- Users
+- Groups
+- Projects
+- Group memberships
+- Project memberships
+
+Once all the necessary resources are fetched, Teleport pushes them to the
+Access Graph, ensuring that it remains updated with the latest information from your GitLab instance.
+
+### Importing resources
+
+Teleport Policy’s Access Graph feature delves into the resources imported and their relationships, crafting a
+graphical representation thereof.
+
+
+## Prerequisites
```

Comment:
Do we know what is the lowest supported GitLab self-hosted version?

Reply:
Gitlab v4 was introduced in Gitlab version 6 and v3 was removed in Gitlab 9.